pv_site_api/main.py

```diff
@@ -182,19 +195,48 @@ def post_pv_actual(
 
 # Comment this out, until we have security on this
 # # put_site_info: client can update a site
-# @app.put("/sites/{site_uuid}")
-# def put_site_info(site_info: PVSiteMetadata):
-#     """
-#     ### This route allows a user to update site information for a single site.
-#
-#     """
-#
-#     if is_fake():
-#         print(f"Successfully updated {site_info.dict()} for site {site_info.client_site_name}")
-#         print("Not doing anything with it (yet!)")
-#         return
-#
-#     raise Exception(NotImplemented)
+@app.put("/sites/{site_uuid}")
+def put_site_info(
+    site_uuid: str,
+    site_info: PVSiteMetadata,
+    session: Session = Depends(get_session),
+    auth: Auth = Depends(auth),
+):
+    """
+    ### This route allows a user to update a site's information.
+
+    """
+
+    if is_fake():
+        print(f"Successfully updated site {site_uuid} with {site_info.dict()}")
+        print("Not doing anything with it (yet!)")
+        return
+
+    # @TODO: get client corresponding to auth
```

Comment:
```suggestion
    # @TODO: get client corresponding to auth
    # See: https://github.com/openclimatefix/pv-site-api/issues/90
```
I made a placeholder issue so that those TODO can point to it.